package.json

```diff
@@ -36,9 +36,10 @@
 		"eject": "react-scripts eject",
 		"prettier-check": "prettier --check src/**/*.ts{,x}",
 		"format": "prettier --write src/**/*.ts{,x}",
+		"format-modules": "prettier --write src/pages/modules/**/*.ts{,x}",
```

Comment:
why add this? Isn't this covered by the script above it?

Reply:
You are right, I was going to remove with the logs when I add the test cases